schema.sql

```diff
@@ -7,5 +7,6 @@ CREATE TABLE animals (
     escape_attempts INT,
     neutered        BOOLEAN,
     weight_kg       DECIMAL(4,2),
+    species         VARCHAR(100),
```

Comment:
Please be sure to use the ALTER statement in order to add the new property of species rather than adding it manually to your table 😁